bin/build_env.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
There are a bunch of tools that we could use, except configman is using identifiers that aren't valid bash identifiers, so none of them work because they're all validating the environment variable keys before throwing them willy-nilly into the environment.

Thus, we had to roll our own. Yay.